DigitisingSC/DHSC-public-website
# Feature/dhsc 128 decision tree tool

- Adds dhsc_result_viewer module
- Adds new content type for Recommendation and Supplier

Test (Self assessment)

1. Import this DB 
[dhsc_decison_tree.gz](https://github.com/DigitisingSC/DHSC-public-website/files/11686093/dhsc_decison_tree.gz)

2. Complete self assessment tool https://dhsc-public-website.ddev.site/form/self-assessment-tool giving different answers and check results return appropriate Recommendation nodes. 

3. Fill in all questions with Yes answers and check results contain additional text. Pulled form /admin/result-summary-settings
